stories/Components/UIcomponents/Cards/BioCard/BioCard.jsx

```diff
@@ -62,9 +60,7 @@ export const BioCard = ({
           )}
 
           {args.size === 'medium' && (
-            <a
-              id="bio-card"
-              className={['bio-card medium', `${hover_color_options[`${args.hovercolor}`]}`].join(' ')}
+            <a className={['bio-card medium', `${hover_color_options[`${args.hovercolor}`]}`].join(' ')}
```

Comment:
@pallavikadam-srijan There is whitespace in class attribute. Kindly use the provided solution of `cls` to add/toggle classes.